wg-charter.html

```diff
@@ -0,0 +1,387 @@
+            that can be translated or compiled into platform-specific instructions</li>
+        </ol>
+
+        <p>
+            The API will not be restricted to any particular platform technology. Instead, it will
+            be generic enough to be implemented on top of modern GPU libraries, such as
+            Microsoft's Direct3D 12, Apple's Metal, and Khronos's Vulkan.
+        </p>
+
+        <p>
+            Note that the majority of the input to this Working Group will come
+            directly from the <a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a>.
+            No major development will happen in the Working Group itself. Instead, the Community
+            Group will be driving the technical work.
+        </p>
+
+        <div>
+          <h3>Success Criteria</h3>
+          <p>In order to advance to  <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have at least two independent implementations of each of feature defined in the specification.</p>
+          <p>The WebGPU specification should contain a section detailing any <a href='https://www.w3.org/TR/security-privacy-questionnaire/'>known security</a>, fingerprinting, and privacy implications, and suggested <a href='https://w3c.github.io/perf-security-privacy/'>mitigation strategies</a> for implementers, Web authors, and end users. The group should not publish a specification if acceptable mitigation strategies cannot be found.</p>
+          <p>The WebGPU specification should contain a section describing known impacts on accessibility to users with disabilities, ways the specification features address them, and recommendations for minimizing accessibility problems in implementation.</p>
+          <p>Normative specification changes are generally expected to have a corresponding set of tests, either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the proposed update.</p>
+        </div>
+
+        <h2>Out of Scope</h2>
+        <p>
+            The scope of work is restricted to the development of a programming
+            interface between the Web Platform and modern 3D graphics and
+            computation capabilities present on native system platforms. The work
+            will not define hardware features or algorithms, and the interface it
+            defines is not intended to be directly exposed by a GPU driver.
+        </p>
+
+      </section>
+
+
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <div id="normative">
+          <h3>
+            Normative
+          </h3>
+          <dl>
+          <dt>WebGPU API</dt>
+
+          <dd>
+              An API as described by the <a
```

Comment:
+1. Especially, since when I looked for a description of what this API is **in the CG charter** and found:

> **GPU for the Web Specification**
      An API as described above.

I'd love to replace this with a summary of what the deliverable is, e.g., from [the spec's abstract](https://gpuweb.github.io/gpuweb/): "WebGPU exposes an API for performing operations, such as rendering and computation, on a Graphics Processing Unit."